test/app-luatest/fix_lua_loader_error_handling_test.lua

```diff
@@ -0,0 +1,24 @@
+local t = require('luatest')
+local treegen = require('test.treegen')
+
+local g = t.group()
+
+local luadir
+local module = 'syntax'
+
+g.before_all(function()
+    treegen.init(g)
+    treegen.add_template(g, ('^%s%%.lua$'):format(module), [[?syntax error?]])
+    luadir = treegen.prepare_directory(g, {('%s.lua'):format(module)})
+end)
+
+g.after_all(function(g)
+    treegen.clean(g)
+end)
+
+g.test_loader_error_handling = function()
+    package.path = ('%s/?.lua'):format(luadir)
```

Comment:
It affects other test files if several tests are run using `luatest` directly (w/o test-run).

Reply:
Thanks! Personally, I would prefer more isolated runs (using `luatest.server`, for example), but I think that this solution is acceptable too.